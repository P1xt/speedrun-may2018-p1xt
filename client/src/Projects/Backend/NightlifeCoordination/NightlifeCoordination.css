.NightlifeCoordination {
  text-align: left;
  margin-top: 40px; }
  .NightlifeCoordination > h1 {
    text-transform: none; }
