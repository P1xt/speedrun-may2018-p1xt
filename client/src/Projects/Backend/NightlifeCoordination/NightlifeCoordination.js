import React from "react";
import { Container, Row, Col } from 'reactstrap';

import "./NightlifeCoordination.css";

const NightlifeCoordination = () => {
  return (
    <div className="NightlifeCoordination">
      <Container>
        <Row>
          <Col>
            <h1>Nightlife Coordination</h1>
          </Col>
        </Row>
      </Container>
    </div>
  );
};

export default NightlifeCoordination;
