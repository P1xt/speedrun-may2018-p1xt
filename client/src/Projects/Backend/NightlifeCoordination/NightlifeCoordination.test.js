import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

import NightlifeCoordination from "./NightlifeCoordination";

describe("<NightlifeCoordination />", () => {
  it("renders without crashing", () => {
    shallow(<NightlifeCoordination />);
  });
  it("renders correctly", () => {
    const tree = renderer.create(<NightlifeCoordination />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
