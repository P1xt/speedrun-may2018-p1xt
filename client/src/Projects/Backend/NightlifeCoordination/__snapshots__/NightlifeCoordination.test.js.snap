// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NightlifeCoordination /> renders correctly 1`] = `
<div
  className="NightlifeCoordination"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <h1>
          Nightlife Coordination
        </h1>
      </div>
    </div>
  </div>
</div>
`;
